src/test/java/com/courtcanva/ccfranchise/mappers/OrderMapperTest.java

```diff
@@ -0,0 +1,23 @@
+package com.courtcanva.ccfranchise.mappers;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Better to [disable wildcard import](https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports)